server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -38,7 +39,21 @@ import io.delta.sharing.server.util.JsonUtils
 class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
 
   def shouldRunIntegrationTest: Boolean = {
-    sys.env.get("AWS_ACCESS_KEY_ID").exists(_.length > 0)
+    sys.env.get("AWS_ACCESS_KEY_ID").exists(_.length > 0) &&
+      sys.env.get("AZURE_TEST_ACCOUNT_KEY").exists(_.length > 0) &&
+      sys.env.get("GOOGLE_APPLICATION_CREDENTIALS").exists(_.length > 0)
+  }
+
+  private def buildGoogleServiceAccountKeyFile: Unit = {
+    // Only write tmp file if the key content is defined, otherwise the key is loaded directly from
+    // the key file path in GOOGLE_APPLICATION_CREDENTIALS.
+    if (sys.env.get("GOOGLE_SERVICE_ACCOUNT_KEY").exists(_.length > 0)) {
+      val serviceAccountKey = sys.env("GOOGLE_SERVICE_ACCOUNT_KEY")
+      val fileWriter = new FileWriter(new File("/tmp/google_service_account_key.json"))
```

Comment:
nit: Since we set the env `GOOGLE_APPLICATION_CREDENTIALS`, it's better to read it from sys.env rather than using the hard code file path.